src/main/java/lotto/controller/LottoController.java

```diff
@@ -7,24 +7,27 @@
 import lotto.domain.Lotteries;
 import lotto.domain.Lotto;
 import lotto.domain.LottoNumber;
+import lotto.domain.PurchaseInfo;
 import lotto.domain.RankingResult;
 import lotto.view.InputView;
 import lotto.view.ResultView;
 
 public class LottoController {
 
+	private final static InputView inputView = new InputView();
```

Comment:
사소하지만 키워드 순서는 static - final 순이 관례이며
static final을 선언한 캐싱한 객체는 변수명을 대문자로 작성해주시면 좋을 것 같네요 😄 
```suggestion
	private static final InputView INPUT_VIEW = new InputView();
```